config/routes.rb

```diff
@@ -1,6 +1,8 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+# frozen_string_literal: true
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+Rails.application.routes.draw do
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] In order not to expose unnecessary endpoints that are not yet created, I suggest you create only the index and show routes instead of having all the RESTFUL URLs that are not yet created. Hint:

```suggestion
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```